$NetBSD: patch-ab,v 1.3 2013/01/20 06:53:53 mef Exp $

Convert install path from fvwm to fvwm2
This can not be done with 
CONFIGURE_ARGS+=        --datarootdir=${PREFIX:Q}/share/fvwm2
this will install fvwm2/fvwm.

--- configure.orig	2013-01-06 10:03:11.000000000 +0900
+++ configure	2013-01-06 10:08:01.000000000 +0900
@@ -3179,7 +3179,7 @@ fi
 
 
 # Define the identity of the package.
- PACKAGE=fvwm
+ PACKAGE=fvwm2
  VERSION=${version}
 
 
